gallery/src/components/NxNexusPageHeader/NxNexusPageHeaderPage.tsx

```diff
@@ -0,0 +1,151 @@
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="nx-table-row">
+            <td className="nx-cell">productInfo</td>
+            <td className="nx-cell">object</td>
+            <td className="nx-cell">Yes</td>
+            <td className="nx-cell">
+              An object containing at least one field: <code className="nx-code">name</code>, the product
+              name to display in the header. Additionally, this object may contain
+              a <code className="nx-code">meta</code> field which might contain information about the license
+              (as it does in IQ), or other information about the product (e.g. beta, pre-release)
+              and a <code className="nx-code">version</code> field holding the product's version number to display.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell">homeLink</td>
+            <td className="nx-cell">string</td>
+            <td className="nx-cell">No</td>
+            <td className="nx-cell">
+              A URL (typically relative) that navigates to the home page of the application. If this prop is
+              present, the logo in the header will be a link to this home page.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell">logoPath</td>
+            <td className="nx-cell">string</td>
+            <td className="nx-cell">No</td>
+            <td className="nx-cell">
+              The path to the image file that will display as the logo. If no URL is provided the default white hexagon
+              on grey background will appear.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell">links</td>
+            <td className="nx-cell">array</td>
+            <td className="nx-cell">No</td>
+            <td className="nx-cell">
+              An optional array of objects describing navigation links to display in the middle section of the header.
+              Each link object should contain a <code className="nx-code">name</code> to be displayed and
+              a <code className="nx-code">href</code> for the link. Additionally, at most one link should include
+              a <code className="nx-code">current</code> flag set to true indicating that it should be styled as
+              the currently active link.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell">children</td>
+            <td className="nx-cell">ReactNode</td>
+            <td className="nx-cell">No</td>
+            <td className="nx-cell">
+              Optional additional JSX content that will be displayed at the right end of the header.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell">meta</td>
```

Comment:
We've already got this described in the `productInfo` area above; this row is basically just duplicating that.

Reply:
> We've already got this described in the `productInfo` area above; this row is basically just duplicating that.

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/a0d95a8604c8edbc642ea722d5f47c9eab0b8966